src/Microsoft.AspNetCore.Mvc.Core/ApiBehaviorOptions.cs

```diff
@@ -52,5 +70,43 @@ public class ApiBehaviorOptions
         /// that are bound from form data.
         /// </summary>
         public bool SuppressConsumesConstraintForFormFileParameters { get; set; }
+
+
+        public bool AllowUseProblemDetailsForClientErrorResponses
+        {
+            get => _allowUseProblemDetailsForClientErrorResponses.Value;
+            set => _allowUseProblemDetailsForClientErrorResponses.Value = value;
+        }
+
+        public IDictionary<int, Func<ActionContext, ProblemDetails>> ProblemDetailsFactory { get; } = new Dictionary<int, Func<ActionContext, ProblemDetails>>
+        {
+            [400] = _ => new ProblemDetails
+            {
+                Type = "https://tools.ietf.org/html/rfc7231#section-6.5.1",
```

Comment:
Are you dead set on these URIs?

I (and [Zend Framework](https://github.com/zendframework/zend-problem-details/blob/1401582c7f2107dd32ce7dcc24642085b518085b/src/ProblemDetailsResponseFactory.php#L394), that I know of) use `https://httpstatus.es/{statusCode}` URIs for generic status code details. IMHO, it's a tad nicer than just linking to the RFC.

Reply:
> Are you dead set on these URIs?

Not particularly. I can find out if we could use the `httpstatuses.com` url here. My only concern is that domain is owned by a private entity (`Registrant Organization: WebpageFX`) that I've never heard of.

Reply:
The project lives at https://github.com/rmaake1/httpstatuses if that helps 🤷‍♂️

Reply:
cc @glennc